### PR TITLE
README

### PR DESCRIPTION
**Pull request by tegamax**
Please approve my PR, I have improved a few things in the document. Please let me know if you need anything else on there.

Thanks,
Tega

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,34 @@
...
+[![Forks](https://img.shields.io/github/forks/tuladhar/multi-threaded-google-ban-checker)](https://github.com/tuladhar/multi-threaded-google-ban-checker/network/members)
+[![Watchers](https://img.shields.io/github/watchers/tuladhar/multi-threaded-google-ban-checker)](https://github.com/tuladhar/multi-threaded-google-ban-checker/watchers)
+[![License](https://img.shields.io/github/license/tuladhar/multi-threaded-google-ban-checker)](https://github.com/tuladhar/multi-threaded-google-ban-checker/blob/main/LICENSE)
+
 NOTE:
 This is not a Google account, domain or advertisements ban checker
 
+## Table of Contents
+* [Installation](#Installation)
+* [How it works?](#How it works?)
```
Comment (tuladhar) [inline README.md:18]: Link is broken.

**Review by tuladhar — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,16 +1,34 @@
...
+[![Open Pull Requests](https://img.shields.io/github/issues-pr-raw/tuladhar/multi-threaded-google-ban-checker)](https://github.com/tuladhar/multi-threaded-google-ban-checker/pulls)
+[![Last Commit](https://img.shields.io/github/last-commit/tuladhar/multi-threaded-google-ban-checker)](https://github.com/tuladhar/multi-threaded-google-ban-checker/commits/main)
+[![Stars](https://img.shields.io/github/stars/tuladhar/multi-threaded-google-ban-checker)](https://github.com/tuladhar/multi-threaded-google-ban-checker/stargazers)
+[![Forks](https://img.shields.io/github/forks/tuladhar/multi-threaded-google-ban-checker)](https://github.com/tuladhar/multi-threaded-google-ban-checker/network/members)
+[![Watchers](https://img.shields.io/github/watchers/tuladhar/multi-threaded-google-ban-checker)](https://github.com/tuladhar/multi-threaded-google-ban-checker/watchers)
+[![License](https://img.shields.io/github/license/tuladhar/multi-threaded-google-ban-checker)](https://github.com/tuladhar/multi-threaded-google-ban-checker/blob/main/LICENSE)
+
```
Comment (tegamax) [inline README.md:12]: these are not the type of links you think. they are API requests to  your repository stats
Screen Shot 2020-10-19 at 12 45 44 PM

File: README.md (hunk 1 above)
Comment (tegamax) [reply README.md:18]: these are also referenced in what we have on the same page, so there is no need to put a link to the same page.
Screen Shot 2020-10-19 at 12 46 09 PM

**Review by tuladhar — APPROVED** [review]
Looks good! Merging.

**Pull request merged**